Network/XmlRpc/DTD_XMLRPC.hs

```diff
@@ -33,6 +34,7 @@ data Value_ = Value_Str String
             | Value_Base64 Base64
             | Value_Struct Struct
             | Value_Array Array
+            | Value_Nil Nil
```

Comment:
At first I wondered about whether there needs to be a `Nil` value inside the `Value_Nil` constructor but I think I see the purpose of it now.